docs/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/is_hvac_sys_cooling_type_none.md

```diff
@@ -1,23 +1,19 @@
 # is_hvac_sys_cooling_type_none  
 
-**Description:** Returns TRUE if the HVAC system cooling type is None or Null. Returns FALSE if the HVAC system has anything other than None or Null for the cooling type or if it has more than 1 or no cooling system.   
+**Description:** Returns TRUE if the HVAC system cooling type is None or Null. Returns FALSE if the HVAC system has anything other than None or Null for the cooling type.   
 
 **Inputs:**  
 - **B-RMR**: To evaluate if the hvac system is modeled with None or null for cooling.   
```

Comment:
It is a little odd that the documentation is written for the Baseline RMR even though it would equally apply to the other RMR.  It would seem more natural to me to, for example, pass `rmi` and `hvac_id` as the arguments.

But I assume that this function will only be used in the context of the Baseline RMR, so I suppose it doesn't matter much.

Reply:
Yes, that is the intent. The intent of these functions is to find out which baseline system type the baseline model is using.